proto/query.proto

```diff
@@ -973,3 +973,27 @@ message TransactionMetadata {
   int64 time_created = 3;
   repeated Target participants = 4;
 }
+
+
+// TableType represents the type of table requested.
+enum TableType {
+  VIEWS = 0;
+  TABLES = 1;
+  ALL_TABLES_VIEWS = 2;
```

Comment:
can be just `ALL`?
that way if we add any other type of objects in the future, the name doesn't become obsolete.